src/dotnet/APIView/APIView/NamespaceAPIV.cs

```diff
@@ -27,7 +27,7 @@ public class NamespaceAPIV
         /// <param name="symbol">The symbol representing the namespace.</param>
         public NamespaceAPIV(INamespaceSymbol symbol)
         {
-            this.Name = symbol.Name;
+            this.Name = symbol.ToDisplayString();
```

Comment:
Hm, aren't they still going to be nested?

Reply:
Yeah, they still are for now - I was planning to undo nesting in the rendering in a later fix

Reply:
Don't do it as part of this PR.